wiki/Getting-Started-UIKit.md

```diff
@@ -0,0 +1,239 @@
+    private let finishButton = UIButton()
+
+    required init(with email: String) {
+        self.email = email
+        super.init(nibName: nil, bundle: nil)
+
+        view.backgroundColor = .systemGray5
+
+        finishButton.setTitle("Finish", for: .normal)
+        finishButton.setTitleColor(.systemBlue, for: .normal)
+        finishButton.addTarget(self, action: #selector(finishPressed), for: .touchUpInside)
+        finishButton.accessibilityIdentifier = "finish"
+
+        view.addSubview(finishButton)
+
+        finishButton.translatesAutoresizingMaskIntoConstraints = false
+        finishButton.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        finishButton.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+    }
+
+    required init?(coder: NSCoder) { nil }
+
+    @objc private func finishPressed() {
+        proceedInWorkflow(email)
+    }
+
+    func shouldLoad() -> Bool {
+        return email.contains("@wwt.com")
+    }
+}
+```
+### Let's talk about what is going on with these view controllers.
+#### **What's this `shouldLoad()`?**
+<details>
+
+It is part of the [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html) protocol. It has default implementations created for your convenience but is still implementable if you want to control when a [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html) should load in the work flow.  It is called after `init` but before `viewDidLoad()`.
+</details>
+
+## Launching the [Workflow](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/Workflow.html)
+Next, we create a [Workflow](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Classes/Workflow.html) that is initialized with our [FlowRepresentable](https://gitcdn.link/cdn/wwt/Workflow/faf9273f154954848bf6b6d5c592a7f0740ef53a/docs/Protocols/FlowRepresentable.html)s and launch it from a view controller that is already loaded onto the screen (in our case, the 'ViewController' class provided by Xcode).
+
+```swift
+import UIKit
+import Workflow
+
+class ViewController: UIViewController {
+    private let launchButton = UIButton()
+
+    init() {
+        super.init(nibName: nil, bundle: nil)
+
+        launchButton.setTitle("Launch Workflow", for: .normal)
+        launchButton.addTarget(self, action: #selector(launchWorkflow), for: .touchUpInside)
+
+        view.addSubview(launchButton)
+
+        launchButton.translatesAutoresizingMaskIntoConstraints = false
+        launchButton.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        launchButton.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
Completely wiping out the ViewController from the storyboards may be going too far with Programmatic?  I'm not sure, but initially on copy pasting my way through the tutorial this fatalError is thrown because this view is no longer loadable from a storyboard.

I suppose this is dependent on if people who do programmatic fully configuring their projects every time to be without the storyboard or if they simple leverage the existing storyboard starting view and then customize from there.